### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Settings_Application_Keyboard : Incorrect keyboard navigation provided for the 'Tab' controls.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/AzureStorageExplorer#3501]
GitHubTags: #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yMAS;#A11YTCS;#Azure Tools Storage Explorer;#WCAG2.1.1;#MAC;#A11ySev2;#Benchmark;#Keyboard;#FTP;#BM-Azure Tools Storage Explorer-MAC-Aug2020;

### Environment Details:
Application Name: Azure Tool Storage Explorer
Application version: 1.15.0
AzCopy Version: 10.6.0
OS Version: macOS Catalina Version 10.15.6
Screen Reader: Voiceover

### Repro Steps:
1. Launch Storage Explorer
2. Navigate to Setting menu on left pane and click on it.
3. Using keyboard, navigate through the tab controls such as "Application", etc.
4. Try navigating using the arrow keys within the tab controls.

### Actual:
When keyboard users navigate through the tab controls such as "Application", "Data Explorers", etc. they have to navigate using the Tab key and not arrow. The native keyboard navigation for the "Tab" controls is using the up and down arrow but here the keyboard users have to navigate using the Tab key which is incorrect.

This issue is observed in the below scenarios:
1. Settings - Data Explorers
2. Settings - Transfers

### Expected:
For keyboard users the up and down arrow keys should be used to navigate within the "Tab" control and not the Tab key.

### User Impact:
If proper keyboard navigation is not provided for the controls present on the screen then the keyboard users will get confused and they will have difficulty in interacting with the controls.

### Attachment:
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-manai_microsoft_com/EfPb5ZfX9K1Et15uNHWc730BJ7XK2LmV7-hUdj-IyIJWnQ?e=1YZH3k

**Comment by JasonYeMSFT** [comment]
@MayuriNaik Could you please verify and close this issue if it is fixed?

**Comment by MayuriNaik** [comment]
The keyboard users are now able to use arrow keys to navigate navigate within the "Tab" control. The bug is fixed and hence closing the bug. Verified in application version Version: 1.16.0-dev AzCopy Version: 10.6.0

**Comment by Ashutoshsahoo164** [comment]
GitHubTags: #Closed
